.env.tmp

```diff
@@ -61,6 +61,17 @@ IOTHUB_DEPLOYMENT_TARGET_CONDITION=""
     # IOTHUB_DEPLOYMENT_TARGET_CONDITION="tags.environment='dev'"
     # IOTHUB_DEPLOYMENT_TARGET_CONDITION="tags.building=9 and tags.environment='test'"
 
+
+#
+# DEVICE_TAGS
+# 
+
+DEVICE_TAGS=""
```

Comment:
Perhaps these should go under the solution section? Since this a solution subcommand

Reply:
Good point, moved it